packages/cli/src/config/index.ts

```diff
@@ -15,6 +15,7 @@ if (inE2ETests) {
 	process.env.N8N_LOG_LEVEL = 'silent';
 	process.env.N8N_PUBLIC_API_DISABLED = 'true';
 	process.env.SKIP_STATISTICS_EVENTS = 'true';
+	process.env.N8N_SECURE_COOKIE = 'false';
```

Comment:
Maybe we could set it to true in tests so it corresponds the most common config? Or leave it out so the default value gets tested

Reply:
we unfortunately can't to do that right now because `superagent` doesn't handle secure cookies, which breaks every integration test that updates the cookie in response.